Doc/library/ensurepip.rst

```diff
@@ -122,6 +123,12 @@ Module API
    *verbosity* controls the level of output to :data:`sys.stdout` from the
    bootstrapping operation.
 
+   *prefix* specifies the directory prefix to use when installing.
+
+   .. versionchanged:: 3.13
```

Comment:
```suggestion
   .. versionadded:: 3.13
```

Reply:
Are you sure? I thing these directives apply to the object containing them, i.e. the file for a module, a function definition block for a function. versionadded inside a function gives info about when the function was added to the module.